### PR TITLE
(CONT-256) Removing outdated code

### PR DESCRIPTION
**Pull request by LukasAud**
Prior to this commit, the firewall modules had multiple instances of outdated code.

This commit aims to clean-up some of that code to keep the module clean and readable.

### CODE REVIEW

File: manifests/linux/debian.pp [hunk 1]
```diff
@@ -25,39 +25,16 @@
   $package_name   = $firewall::params::package_name,
   $package_ensure = $firewall::params::package_ensure,
 ) inherits ::firewall::params {
-  if $package_name {
```
Comment (david22swan) [inline manifests/linux/debian.pp:28]: You still need the `if $package_name`

**Review by david22swan — CHANGES_REQUESTED** [review]
Oe small comment and a syntax fix.
Otherwise looks good

File: manifests/linux/debian.pp (hunk 1 above)
Comment (LukasAud) [reply manifests/linux/debian.pp:28]: re-implemented

**Review by david22swan — APPROVED** [review]
LGTM

**Pull request merged**